assign_policy.yml

```diff
@@ -0,0 +1,17 @@
+---
+- hosts: hosts
+
+  tasks:
+  - name: Assign policy on an OpenShift provider in ManageIQ
+    manageiq_policy:
+      entity: '{{ entity }}'
+      entity_name: '{{ entity_name }}'
+      resource: '{{ resource }}'
+      resource_name: '{{ resource_name }}'
+      action: '{{ action }}'
```

Comment:
@dkorn,

I personally do not like the "action" calls.  I'd prefer separate actions move to different ansible modules.  With every action added the code paths increase as well as your parameters.  Then it becomes unclear which parameters are required with which actions.  Separate actions moved into their own modules makes for modular code, loose coupling, and simpler interfaces for the consumers.